### ISSUE TITLE
/W and /w all masters

### ISSUE THREAD

**Issue by tphinney** [tphinney/science-gothic#55]
The last of the basic English-Latin alphabetics to fill in!

**Comment by tphinney** [comment]
- It looks like there is less optical compensation in the joins than in the /V in the :Blk weights. I consider that to be working really well in the /V, and I expect the /W needs at least as much, perhaps more. In the :Blk /W on the left stroke I see about half as much thinning as I go down the stroke from the top, compared to the /V at the same height (like 20 units instead of 40).

- In the :Cnd, a few units more thinning in the joins as well. It is closer than the :Blk.

- Have not checked other styles; let me know if more feedback would be helpful.

I would be happy to do some surgery to enact changes, if you like.

**Comment by tphinney** [comment]
I'm separating cap /W from lowercase /w in the issue tracker.
I just did some cap /W edits for my next submit.
Not sure it is done, but I think it is somewhat better.

**Comment by servantb** [comment]
Is there a reason for stress reversal of right half of /W in all but the :Med, :LtEtc, and :CtrEtc? I know I've seen it done before, but I'm not sure here is the place. If we want to keep it consistent, I feel like the stress should stay traditional.

A couple more observations:
Right most stroke is too wide at the top in :Exp.
The :BlkExp is too heavy.
Stroke angles should be closer to equal in :Ctr, :CndCtr, and :BlkCndCtr.

**Comment by servantb** [comment]
The /W :Cnd and :Exp are too flared on the outside terminals, and don't follow the traditional ductus (thick thin thick thin), which is fine in the black masters, but not in the medium. I edited them to be more monoline, and also moved the vertices slightly toward the outside in the :Cnd.

I tweaked the :BlkCnd and :BlKExp similarly to the above, but to a lesser degree.

Edited the :BlkExpCtr to match width of :BlkExp, reduced flare.
